crates/holochain_metrics/src/test.rs

```diff
@@ -9,6 +10,60 @@ fn metrics_none() {
     assert!(matches!(config, HolochainMetricsConfig::Disabled));
 }
 
+#[cfg(feature = "influxive")]
+#[tokio::test(flavor = "multi_thread")]
+async fn metrics_influxive_file() {
+    let temp_dir = tempfile::TempDir::new().unwrap();
+    let filepath = temp_dir
+        .path()
+        .join(std::path::PathBuf::from("metrics.influx"));
+
+    let config = HolochainMetricsConfig::from_env(
+        temp_dir.path(),
+        HolochainMetricsEnv::InfluxiveFile {
+            filepath: filepath.as_path().to_str().unwrap().to_string(),
+        },
+    );
+    assert!(matches!(
+        config,
+        HolochainMetricsConfig::InfluxiveFile {
+            writer_config: _,
+            otel_config: _
+        }
+    ));
```

Comment:
Is there a reason why this shouldn't be:

```suggestion
    assert!(matches!(config, HolochainMetricsConfig::InfluxiveFile { .. }));
```

Reply:
Not really. Will fix.